src/modules/AutoinstClass.rb

```diff
@@ -98,7 +98,6 @@ def class_dir=(newdir)
       AutoinstConfig.classDir = newdir
       @classDir = newdir
       @classPath = File.join(@classDir, @class_file)
-      newdir
```

Comment:
Although it modifies the behavior, it is not needed to return `newdir`. So it is OK.

Reply:
in fact it does not modify behavior as attr_writters does not return different value. Try youself:
```
irb(main):001:0> class C;def a=(v); @a = v; return 256;end;end
=> :a=
irb(main):002:0> c = C.new
=> #<C:0x00005651398f54e0>
irb(main):003:0> b = c.a = 15
=> 15
irb(main):004:0> b
=> 15
```

Reply:
Interesting... :thinking: